src/control/lib/spdk/spdk.go

```diff
@@ -65,13 +66,13 @@ func Rc2err(label string, rc C.int) error {
 // EnvOptions describe parameters to be used when initializing a processes
 // SPDK environment.
 type EnvOptions struct {
-	MemSize      int      // size in MiB to be allocated to SPDK proc
-	PciWhiteList []string // restrict SPDK device access
-	DisableVMD   bool     // flag if VMD devices should not be included
+	MemSize        int      // size in MiB to be allocated to SPDK proc
+	PciIncludeList []string // restrict SPDK device access
+	DisableVMD     bool     // flag if VMD devices should not be included
 }
 
-func (o EnvOptions) toC() (opts *C.struct_spdk_env_opts, cWhiteListPtr *unsafe.Pointer, err error) {
-	opts = new(C.struct_spdk_env_opts)
+func (o *EnvOptions) toC(log logging.Logger) (*C.struct_spdk_env_opts, *unsafe.Pointer, error) {
```

Comment:
As a potential future optimization, it occurs to me that it would be better to return a cleanup closure that takes care of freeing any C-allocated memory in this function. That way the caller doesn't need to know the details, and if we eventually need to allocate other memory here, it all stays encapsulated in this function.

Reply:
fixed